java-checks/src/test/files/checks/StaticImportCountCheck/MixedStandardAndStaticImports.java

```diff
@@ -0,0 +1,9 @@
+import java.util.Date;
+import java.util.Collections;
+import java.util.List;
+
+import static java.lang.Math.*;
+import static java.util.Collections.*;
+import static com.myco.corporate.Constants.*;
+import static com.myco.division.Constants.*;
+import static com.myco.department.Constants.*;  // Noncompliant
```

Comment:
You want to report the whole clause, not the whole line so you can check you report starting from column 1 to the column of the semicolon.